### ISSUE TITLE
Scheduling notifications is too slow

### ISSUE THREAD

**Issue by yjukaku** [DrewDahlman/Phonegap-LocalNotification#17]
Notifications take a very long time to schedule, especially if you want to use up all 64 allowed local notifications. This could be solved by allowing multiple notifications to be scheduled at the same time.

**Comment by DrewDahlman** [comment]
This is true but is a case by case scenario. If you have code you would like to provide that gives an example of this use feel free to fork and contribute. Maybe adding a new schedule method that is for bulk scheduling would be a good solution...
